examples/analysis_workflow.py

```diff
@@ -0,0 +1,37 @@
+"""Creating an example for the new trajectory module."""
+
+import MDAnalysis
+
+from dynsight.trajectory import Trj
+
+
+def main() -> None:
+    """Creating an example for the new trajectory module."""
+    # Create a Trj object
+    universe = MDAnalysis.Universe(
+        "analysis_workflow/oxygens.gro", "analysis_workflow/oxygens.xtc"
+    )
+
+    water_trj = Trj(universe)
+    water_trj.save("analysis_workflow/water_trj")
+
+    # We want for instance compute LENS on this trajectory
+    # This is an Insight object
+    water_lens = water_trj.get_lens(r_cut=7.5)
+
+    # We can do spatial average on the computed LENS
```

Comment:
`# From here, we work with `Insight` objects that contain data from a `Trj` object`?